std/node/_fs/_fs_readlink_test.ts

```diff
@@ -1,18 +1,20 @@
 const { test } = Deno;
 import { readlink, readlinkSync } from "./_fs_readlink.ts";
 import { assertEquals, assert } from "../../testing/asserts.ts";
+import * as path from "../path.ts";
 
 const testDir = Deno.makeTempDirSync();
-const oldname = testDir + "/oldname";
-const newname = testDir + "/newname";
+const oldname = path.join(testDir + "/oldname");
+const newname = path.join(testDir + "/newname");
```

Comment:
What does `path.join()` do when given only a single argument? Do you mean to make the plus character into a comma?

Reply:
of course, edited!
thanks you 👍